### DISCUSSION TITLE
Informational status for Table Cell

### DISCUSSION THREAD

**Discussion by laurynas111** [iTwin/iTwinUI#2360] [Ideas]
Hi,

I'm working with Table component and trying to color rows with red, green and blue colors: https://itwinui.bentley.com/docs/table

Would it be possible to extend DefaultTableCell status property to support informational status?
iTwinUI varaibles seem to define positive, negative, warning and informational color backgrounds - https://itwinui.bentley.com/docs/variables. However, DefaultTableCell accepts all status valyes as properties except for informational one

**Comment by r100-stack** [comment]
Before addressing the question, just curious, what is your goal that you're trying to achieve using the informational status on the Table row? If there are other ways to implement your objective, we can try suggesting alternatives.

This is because we had discussed something slightly similar internally a while back and had landed on the conclusion that the informational status would not supported by default, especially for the use cases where alternatives are available.

**Reply by laurynas111** [reply]
I'm coloring Elements in 3rd model based on the element modification:
Green for added element, red - removed, blue - modified.
I also list all elements in a Table (each table row defines and Element in the 3d model) and need to somehow show same status for each row. I wanted to use same colors in the Table that I use in the 3rd model. However, blue color is missing.

Though, I'm not sure whether I should be trying to "fit" statuses to how Elements are colored in the 3d model.

**Reply by r100-stack** [reply]
Thanks for the clarification. However, during our internal discussion, we were still a little confused about your use-case. E.g. by "informational status for Table Cell", are you referring to the row status (stripe at the left edge of a row) or the cell status (background color of the cell). [Example usage](https://itwin.github.io/iTwinUI/react/?story=table--status-and-cell-icons&theme=dark) of both. Could you further clarify your use-case, preferably accompanied with a screenshot?

In the meanwhile, one of the reasons we don't support the informational color in neither the row nor the cell is because it is too similar to the color of a selected row.

<details><summary>Screenshot</summary>

![image](https://github.com/user-attachments/assets/51c21709-4232-4816-97fe-e9206d8ec59d)

</details> 

Some alternatives to the informational color are to have a separate column for the status (useful for filtering too) and/or use icons in the `TableCell` to denote the statuses (see same link above for examples). These alternatives will also lead to better UX and accessibility since color is not the only indication of the status (e.g. useful for color blind users).
